doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -141,6 +145,7 @@ nRF9160: Asset Tracker v2
     * The conversions of RSRP and RSRQ now use common macros that follow the conversion algorithms defined in the `AT Commands Reference Guide`_.
     * Bootstrapping has been disabled by default to be able to connect to the default LwM2M service AVSystem's `Coiote Device Management`_ using free tier accounts.
     * Added support for full modem FOTA updates for nRF Cloud builds.
+    * The default sample timeout for sample requests that include neighbor cell searches is now 11 seconds.
```

Comment:
It loooks like it's hard-coded to 11 seconds, or can it change somewhere else in the code?

```suggestion
    * The timeout for sample requests that include neighbor cell searches is now 11 seconds.
```

Reply:
It's a fixed value based on data I got from the modem team.

Reply:
Then I think the changelog should not say «default». That would indicate that it's possible to change. Please see suggested change ☝️